src/Components/animation/Animation.js

```diff
@@ -150,6 +154,10 @@ class Animation extends React.Component {
 
                 if (pageY !== undefined) this.dTop = pageY;
                 if (pageX !== undefined) this.dLeft = pageX;
+
+                if (Platform.OS === 'ios') {
+                  pageX = 0;
```

Comment:
It looks like it does not change anything here because we have already used pageX to modify this.dLeft

We should either do it before the assignation or remove it (if it's useless)